### PR TITLE
Custom product Content Type clarification

### PR DESCRIPTION
**Pull request by DominikaK**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | n/a
| Versions      | 3.3

### CODE REVIEW

**Review by reithor — APPROVED** [review]

File: docs/guide/catalog/product.md [hunk 1]
```diff
@@ -43,12 +43,23 @@ silversolutions_eshop:
             - my_custom_product_type
```
Comment (dabrt) [inline docs/guide/catalog/product.md:43]: Should there be a minus before "my_custom_product_type"? 
`yaml` styling interprets it as substraction and colors the line red while we just said that we want to add this identifier to the config file.

File: docs/guide/catalog/product.md [hunk 2]
```diff
@@ -43,12 +43,23 @@ silversolutions_eshop:
...
 ```
 
-Additionally, to enable buying the product in the shop,
-add the following parameter for your custom Content Type's identifier:
+The custom Content Type must mirror the structure of the built-in `ses_product` Content Type,
+including having Fields with the same identifiers,
```
Comment (dabrt) [inline docs/guide/catalog/product.md:47]: Is it "the structure that includes the Fields with the same identifiers", or is it "by including/having the Fields with the same IDs", or would it be just "The custom Content Type must include all the Fields with the same identifiers as the built-in `ses_product` Content Type, ..."?

File: docs/guide/catalog/product.md [hunk 3]
```diff
@@ -43,12 +43,23 @@ silversolutions_eshop:
...
+but you can add other Fields to it.
+
+!!! tip
+
+    To ensure all the Fields are set up correctly, you can copy the built-in Product Content Type
```
Comment (dabrt) [inline docs/guide/catalog/product.md:52]:
```suggestion
    To ensure that all the Fields are set up correctly, you can copy the `ses_product` Content Type
```

File: docs/guide/catalog/product.md [hunk 4]
```diff
@@ -43,12 +43,23 @@ silversolutions_eshop:
...
+    and add your custom Field to the copy.
+
+Additionally add the following parameters for your custom Content Type's identifier:
```
Comment (dabrt) [inline docs/guide/catalog/product.md:55]:
```suggestion
Additionally, add the following parameters for your custom Content Type's identifier:
```

**Review by dabrt — APPROVED** [review]

File: docs/guide/catalog/product.md (hunk 1 above)
Comment (DominikaK) [reply docs/guide/catalog/product.md:43]: That's part of YAML's array syntax: https://www.w3schools.io/file/yaml-arrays/, see for example: https://github.com/ezsystems/ezcommerce-shop/blob/7169ce3dfbafca67f696b2009209d83414207bb8/src/Silversolutions/Bundle/EshopBundle/Resources/config/product_settings.yaml#L3

File: docs/guide/catalog/product.md (hunk 2 above)
Comment (DominikaK) [inline docs/guide/catalog/product.md:47]:
```suggestion
by having Fields with the same identifiers,
```

**Pull request merged**